### PR TITLE
include_sql_query requests unuseful parameters for CSV database driver (#1437)

### LINKED ISSUE #1437: include_sql_query requests unuseful parameters for CSV database driver
(This issue was separated from the issue #1434.)

Version
-------

At least 6.2.48 and later.

Installation method
-------------------

Any.

Expected behavior
-----------------

CSV driver requests only necessary parameters, which are currently `db_type` and `f_dir`.

Actual behavior
---------------

According to a report, Sympa won't let you define an `include_sql_query` without `db_name`, `db_host` and `db_user`.

Steps to reproduce
------------------

See "Steps to reproduce" in #1434 .

Additional information
----------------------

Some parameters are inconsistent in usage across the various contexts of configurations.  Correction will be suggested in subsequent comment.

### PR DESCRIPTION
**Pull request by ikedas**
This may fix #1437.

### CODE REVIEW

**Comment by salaun-urennes1** [comment]
I did a quick test and now I can define a CSV-based datasource

**Comment by ikedas** [comment]
Thanks!

File: src/lib/Sympa/DatabaseDriver/CSV.pm [hunk 1]
```diff
@@ -30,15 +30,17 @@ use warnings;
 use base qw(Sympa::DatabaseDriver);
 
 use constant required_modules    => [qw(DBD::CSV)];
-use constant required_parameters => [qw(f_dir)];
+use constant required_parameters => [qw(db_name)];
```
Comment (mercitahipolito9-lab) [inline src/lib/Sympa/DatabaseDriver/CSV.pm:33]: 09531282011

File: src/lib/Sympa/DatabaseDriver/CSV.pm [hunk 2]
```diff
@@ -30,15 +30,17 @@ use warnings;
 use base qw(Sympa::DatabaseDriver);
```
Comment (mercitahipolito9-lab) [inline src/lib/Sympa/DatabaseDriver/CSV.pm:30]: Wrong translation

**Pull request merged**